packages/babel-helper-replace-supers/src/index.js

```diff
@@ -164,7 +152,7 @@ export default class ReplaceSupers {
    *
    * @example
    *
-   *   _get(CLASS.prototype.__proto__ || Object.getPrototypeOf(CLASS.prototype), "METHOD", this)
+   *   _get(Object.getPrototypeOf(CLASS.prototype), "METHOD", this)
```

Comment:
This should be `_getPrototypeOf`

Reply:
Illustrative, since `Object.getPrototypeOf` is a known algorithm.